### ISSUE TITLE
[BUG] RPM build fails with out of memory

### ISSUE THREAD

**Issue by itkach** [serokell/tezos-packaging#10]
# Description

static build and .deb build are successful but RPM build fails with out of memory error

# To Reproduce
```
[vagrant@nixbox:~/tezos-client]$ nix-build -A babylonnet-rpm-package -o tezos-client-package --arg timestamp $(date  +"%Y%m%d%H%M")
building '/nix/store/6rh2a0wv4bzidzgyfwf2kpm1sk1nck3x-fedora-27-x86_64.nix.drv'...
parsing packages in ./packages_1.xml...
Out of memory!
builder for '/nix/store/6rh2a0wv4bzidzgyfwf2kpm1sk1nck3x-fedora-27-x86_64.nix.drv' failed with exit code 1
error: build of '/nix/store/6rh2a0wv4bzidzgyfwf2kpm1sk1nck3x-fedora-27-x86_64.nix.drv' failed
(use '--show-trace' to show detailed location information)
```
# Expected behavior

builds succeeds 

# Environment
- OS: NixOS VM (VirtualBox) with 2GB of RAM 
- revision: d3686b35b5228f010f3e3479fe85b2bd5ad60791

**Comment by itkach** [comment]
Looks like nix spins up an entire new VM for Fedora and Ubuntu just to produce rpm and deb (which are just compressed archive file formats after, after all). That seems rather wasteful in terms of resources and build time.

**Comment by gromakovsky** [comment]
Isn't it fixed by #15?

**Comment by gromakovsky** [comment]
Okay, it should be fixed by #15. @itkach if the fix does not work for you, please reopen this issue.
